src/Cache/Exception/InvalidArgumentException.php

```diff
@@ -25,3 +25,10 @@
 class InvalidArgumentException extends CakeException implements InvalidArgumentInterface
 {
 }
+// phpcs:disable
+// @deprecated Backwards compatibility alias. Will be removed in 5.0
+class_alias(
+    'Cake\Cache\Exception\InvalidArgumentException',
+    'Cake\Cache\InvalidArgumentException'
+);
```

Comment:
Thanks for adding this. We'll also need a stub class in the old file location that contains a `class_exists('Cake\Cache\Exception\InvalidArgumentException')` in it. This will force the new class & the related alias to be loaded.